.github/workflows/validate-jsons-yamls.yml

```diff
@@ -0,0 +1,92 @@
+#  Copyright 2024 Collate
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#  http://www.apache.org/licenses/LICENSE-2.0
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+name: Validate JSON/YAML
+
+# read-write repo token
+# access to secrets
+on:
+  pull_request_target:
+    types: [labeled, opened, synchronize, reopened]
```

Comment:
should we limit this to certain paths where we have the schemas / yaml examples?

Reply:
done